app/services/katello/pulp3/repository/docker.rb

```diff
@@ -0,0 +1,101 @@
+require 'pulp_docker_client'
+require 'pry'
+
+module Katello
+  module Pulp3
+    class Repository
+      class Docker < ::Katello::Pulp3::Repository
+        def create_remote
+          remote_docker_data = PulpDockerClient::DockerRemote.new(remote_options)
+          response = pulp3_api.remotes_docker_docker_create(remote_docker_data)
+          repo.update_attributes!(:remote_href => response._href)
+        end
+
+        def remote_options
+          if root.docker_tags_whitelist.any?
+            common_remote_options.merge({url: root.url, upstream_name: root.docker_upstream_name, whitelist_tags: root.docker_tags_whitelist})
+          else
+            # TODO: Remove placeholder once whitelist_tags supports empty string
+            common_remote_options.merge({url: root.url, upstream_name: root.docker_upstream_name, whitelist_tags: "placeholder!"})
```

Comment:
Lets go ahead and open  up a issue to track this

Reply:
I'm going to replace `"placeholder!"` with empty string once I un-draft this PR.  The fix in pulp-docker should be out soon.

Reply:
(Taken care of in latest commit)